modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/info/item/provider/test/ObjectEntryInfoItemFieldValuesProviderTest.java

```diff
@@ -173,6 +175,15 @@ public void setUp() throws Exception {
 			LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
 			"oneToManyRelationshipName", false,
 			ObjectRelationshipConstants.TYPE_ONE_TO_MANY, null);
+
+		_originalName = PrincipalThreadLocal.getName();
+
+		PrincipalThreadLocal.setName(TestPropsValues.getUserId());
+	}
+
+	@After
+	public void tearDown() {
+		PrincipalThreadLocal.setName(_originalName);
```

Comment:
@vngoliveira if the `PermissionCheckerMethodTestRule` does not work, please change the code to something similar to [this other test](https://github.com/liferay/liferay-portal/blob/eea2628408e5340d5e4e30e112ed24e443b9a103/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipServiceTest.java#L97-L107), i.e.,

- Also save the `_originalPermissionChecker`;
- We usually call the `PrincipalThreadLocal.setName` and the `PermissionThreadLocal.setPermissionChecker` inside the test method, and not inside the `setUp` method;